qt/widgets/common/src/MantidWSIndexDialog.cpp

```diff
@@ -619,6 +620,18 @@ void MantidWSIndexWidget::onPlotOptionChanged(const QString &plotOption) {
   }
 }
 
+namespace {
+struct LogTestStruct {
```

Comment:
I think this can actually be shortened to:
```cpp
struct LogTestStruct{
	bool isconstantvalue = true;
	double value = std::numeric_limits<double>::quiet_NaN();
};
```

And then initialised [here](https://github.com/mantidproject/mantid/pull/24136/files#diff-4a07fd958032c1bd71425155a110ac9aR659) with
```cpp
LogTestStruct { true, runObj.getLogAsSingleValue(name, Mantid::Kernel::Math::TimeAveragedMean)) }
```